README.rst

```diff
@@ -47,36 +47,75 @@ Geo
 ---
 
 The ``text_scrubber.geo`` module contains functions to normalize geographical data which deal with spelling errors,
-country name variations, etc.:
+country name variations, etc. In the output the ``NormalizedCountryMatch`` and ``NormalizedLocationMatch`` object names
+have been replaced with ``Match`` to make the output more readable.
 
 .. code-block:: python
 
     from text_scrubber.geo import normalize_country, normalize_region, normalize_city
 
-    # Countries
-    normalize_country('Peoples rep. of China')  # [('China', 1.0)]
-    normalize_country('Deutschland')            # [('Germany', 1.0)]
-    normalize_country('st Nevis and Kitties')   # [('Saint Kitts and Nevis', 0.75)]
-    normalize_country('ira')                    # [('Iran', 0.857), ('Iraq', 0.857)]
-
-    # Cities
-    normalize_city('Leibnitz', ['Austria'])    # [('Leibnitz', 'Austria', 1.0)]
-    normalize_city('heidelberg')  # [('Heidelberg', 'Germany', 1.0),
-                                  #  ('Heidelberg', 'South Africa', 1.0),
-                                  #  ('Heidelberg', 'United States', 1.0)]
-    normalize_city('ohioo', ['US'])  # [('Ohio', 'United States', 0.889)]
-    normalize_city('Madri', ['Spain', 'US', 'Brazil'])  # [('Madrid', 'Spain', 0.909),
-                                                        #  ('Madrid', 'United States', 0.909),
-                                                        #  ('Mari', 'Brazil', 0.889)]
-
-    # Regions
-    normalize_region('triangle park', ['US'])   # [('The Triangle Park', 'United States', 1.0)]
-    normalize_region('Fur', ['Denmark'])        # [('Fur', 'Denmark', 1.0)]
-    normalize_region('texel', ['NL'])            # [('Texel', 'Netherlands', 1.0)]
-
-Each of the above normalization functions will return the match score as last entry in the tuple. These scores are
-always between 0.0 and 1.0, where 1.0 is a perfect match. If a known mapping exists, like ``Deutschland`` to
-``Germany``, then the match score will be 1.0.
+    """
+    Countries
+    """
+
+    normalize_country('Peoples rep. of China')
+    # [Match(canonical_name='China', matched_name='Peoples Republic of China', score=1.0)]
```

Comment:
isn't `Match` replaced by `CountryMatch` and `LocationMatch` ? if so can you fix it in the README.md

Reply:
ah i missed that ... cool ...